src/main/java/org/influxdb/impl/InfluxDBImpl.java

```diff
@@ -318,8 +319,9 @@ public QueryResult query(final Query query) {
       call = this.influxDBService.postQuery(this.username,
                                             this.password, query.getDatabase(), query.getCommand());
     } else {
+      String encodedCommand = UrlEscapers.urlFormParameterEscaper().escape(query.getCommand());
```

Comment:
okhttp-3.4.2 use different policies on encoding query parameter and form field.

For query parameter
[HttpUrl.java#L1120](https://github.com/square/okhttp/blob/parent-3.4.2/okhttp/src/main/java/okhttp3/HttpUrl.java#L1120)
```java
    /** Encodes the query parameter using UTF-8 and adds it to this URL's query string. */
    public Builder addQueryParameter(String name, String value) {
      if (name == null) throw new NullPointerException("name == null");
      if (encodedQueryNamesAndValues == null) encodedQueryNamesAndValues = new ArrayList<>();
      encodedQueryNamesAndValues.add(
          canonicalize(name, QUERY_COMPONENT_ENCODE_SET, false, false, true, true));
      encodedQueryNamesAndValues.add(value != null
          ? canonicalize(value, QUERY_COMPONENT_ENCODE_SET, false, false, true, true)
          : null);
      return this;
    }
```
it does not encode some characters, for example ```;```, but influxdb need ```;``` encode into ```%3B```.

For form field
[FormBody.java#L110](https://github.com/square/okhttp/blob/parent-3.4.2/okhttp/src/main/java/okhttp3/FormBody.java#L110)
```java
    public Builder add(String name, String value) {
      names.add(HttpUrl.canonicalize(name, FORM_ENCODE_SET, false, false, true, true));
      values.add(HttpUrl.canonicalize(value, FORM_ENCODE_SET, false, false, true, true));
      return this;
    }
```
it does encode ```;``` and some other characters.